src/arrayMethodLastIndexOf.js

```diff
@@ -4,8 +4,21 @@
  * Implement method lastIndexOf
  */
 function applyCustomLastIndexOf() {
-  [].__proto__.lastIndexOf2 = function(searchElement, fromIndex) {
-    // write code here
+  [].__proto__.lastIndexOf2 = function(searchElement, fromIndex = this.length) {
+    if (this.length === 0 || searchElement === undefined) {
+      return -1;
+    }
+
+    let i = fromIndex >= this.length ? this.length
```

Comment:
you already have `this.length` as default value, why do you add this to the condition?